src/client-config.js

```diff
@@ -16,9 +16,12 @@ window.config = {
   MINIO_BUCKET: 'ocrvs',
   COUNTRY_CONFIG_URL: 'http://localhost:3040',
   // Country code in uppercase ALPHA-3 format
-  COUNTRY: 'FAR',
-  LANGUAGES: 'en,fr',
-  SENTRY: '',
+  COUNTRY: 'MDG',
+  AVAILABLE_LANGUAGES_SELECT: 'fr:Français,en:English,mg:Malagasy',
```

Comment:
AVAILABLE_LANGUAGES_SELECT was this deprecated on 1.4?

Reply:
Great catch! will remove it, thanks @makelicious